config/config.exs

```diff
@@ -12,9 +12,9 @@ config :elixir_boilerplate, ElixirBoilerplateWeb.Endpoint,
   pubsub: [name: ElixirBoilerplate.PubSub, adapter: Phoenix.PubSub.PG2],
   render_errors: [view: ElixirBoilerplateWeb.Errors.View, accepts: ~w(html json)]
 
-config :elixir_boilerplate, ElixirBoilerplate.Gettext, default_locale: "en"
+config :elixir_boilerplate, Corsica, allow_headers: :all
 
-config :elixir_boilerplate, :corsica, allow_headers: :all
+config :elixir_boilerplate, ElixirBoilerplate.Gettext, default_locale: "en"
```

Comment:
Why do we have to change the order in `config/config.exs`?

Reply:
It's purely aesthetic, but it made no sense to me to see the GetText config between `Endpoint` and `Corsica`.